api/translator/src/main/java/com/project/translator/domain/MessageDomain.java

```diff
@@ -0,0 +1,18 @@
+package com.project.translator.domain;
+
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.Collections;
+import java.util.Set;
+
+@Value
+@Builder
+public class MessageDomain {
+    Long id;
+    MessageDomain originalMessage;
```

Comment:
Na podstawie założeń powinien być przekazany tylko id oryginalnej wiadomości, a nie cały obiekt. Prośba o zmianę

Reply:
Zastanawiam się czy warto stworzyć jeszcze dodatkowe dto's dla requestBody / response z kontrolera i mapowac pomiedzy domain i respone. W teorii domain nie powinien byc zwracany i uzywany przez kontrolery.

Reply:
Czy obiekty ...Domain są ważne z punktu widzenia architektury heksagonalnej? Jeżeli tak to zostawiamy tak jak jest teraz i dopisujemy dodatkowe mappery domain <-> dto. Jeżeli nie to wyrzucamy map struct'a i piszemy ręcznie mapowania dto <-> entity.